app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +13,7 @@ class AppServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    private $customAttributes = [];
+    private $customAttributes;
```

Comment:
What is the purpose of this property?

Reply:
Ok, so in the `Validator::replacer` I don't have access to the `Validator`, and because consequently don't have access to he custom attributes names, when they exits. This is a variable that will be filled in the  `Validator::extend` and used in the `Validator::replacer`. It was the way that I managed to do this trick. What do you think?

Reply:
Ok, got it. Honestly, I do not like this approach maybe because we are on the AppServiceProvider and we have properties concerning the extension validator rules. However, I got the point and my suggestion to improve this is to implement a custom Validator (as I've mentioned before during one of our conversations). No problem. I approve it 👍